flocker/control/_protocol.py

```diff
@@ -316,7 +325,7 @@ def noop(self):
 
     @VersionCommand.responder
     def version(self):
-        return {"major": 1}
+        return {"major": 2}
```

Comment:
A nice issue for the debugging epic would probably be "fail with a clear error when mixed protocol versions are encountered".